### PR TITLE
Add loading animations

### LINKED ISSUE #107: add loading animation to all loads
- not the skeleton, but a simpler loading animation for now
- loading for table
- waiting for insert/update/delete

### PR DESCRIPTION
**Pull request by mahos**
Resolves #107 
- add loading animation while waiting for table list
tableListLoading2

- add loading animation while waiting for table content
tableDataLoad

- add animation while waiting for table action (insert/update/delete)
tableActionWaitSmall

### CODE REVIEW

File: src/Components/LoadingAnimation/TableListLoading.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <div className="tableListSkeletons">
+        {[...Array(14)].map((value: undefined, index: number) => {
+          return (
+          <div className="tableSkeleton" key={index}>
+            <div className="tableTierLabelSkeleton"></div>
+            <div className="tableNameSkeleton"></div>
+          </div>
+          )
+        })}
+
```
Comment (Synicix) [inline src/Components/LoadingAnimation/TableListLoading.tsx:23]: Remove this black line

File: src/Components/LoadingAnimation/TableListLoading.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    return (
+      <div className="tableListSkeletons">
+        {[...Array(14)].map((value: undefined, index: number) => {
+          return (
+          <div className="tableSkeleton" key={index}>
+            <div className="tableTierLabelSkeleton"></div>
+            <div className="tableNameSkeleton"></div>
+          </div>
+          )
+        })}
```
Comment (Synicix) [inline src/Components/LoadingAnimation/TableListLoading.tsx:22]: I think that last } should be maybe be on a new line?

File: src/Components/MainTableView/TableContent.tsx [hunk 3]
```diff
@@ -20,10 +21,10 @@ enum PaginationCommand {
...
 
 enum TableActionType {
-  FILTER,
-  INSERT,
-  UPDATE,
-  DELETE
+  FILTER = 'Filter',
+  INSERT = 'Insert',
+  UPDATE = 'Update',
+  DELETE = 'Delete'
```
Comment (Synicix) [inline src/Components/MainTableView/TableContent.tsx:27]: Try to keep this as default autoincrement intergers cause that is vastly more efficeint then comparing strings.

If you want to get the strings, you can write a quick function to if and else the enum then return a string.

File: src/Components/MainTableView/TableView.tsx [hunk 4]
```diff
@@ -530,7 +531,8 @@ class TableView extends React.Component<{token: string, selectedSchemaName: stri
     } else {
       return (
         <div className="loadingArea">
-          <div className="loadingMessage">Loading...</div>
+          <BasicLoadingIcon size={100} />
+          {/* <div className="loadingMessage">Loading...</div> */}
```
Comment (Synicix) [inline src/Components/MainTableView/TableView.tsx:535]: Stale comment?

File: src/Components/LoadingAnimation/TableListLoading.tsx (hunk 2 above)
Comment (mahos) [reply src/Components/LoadingAnimation/TableListLoading.tsx:22]: addressed

File: src/Components/LoadingAnimation/TableListLoading.tsx (hunk 1 above)
Comment (mahos) [reply src/Components/LoadingAnimation/TableListLoading.tsx:23]: addressed

File: src/Components/MainTableView/TableView.tsx (hunk 4 above)
Comment (mahos) [reply src/Components/MainTableView/TableView.tsx]: addressed

File: src/Components/MainTableView/TableContent.tsx (hunk 3 above)
Comment (mahos) [reply src/Components/MainTableView/TableContent.tsx]: addressed

**Pull request merged**